dev/loadgen/pkg/loadgen/executor.go

```diff
@@ -203,22 +204,26 @@ func (w *WsmanExecutor) StopAll(ctx context.Context) error {
 
 	for {
 		resp, err := w.C.GetWorkspaces(ctx, &listReq)
-		if len(resp.Status) == 0 {
-			break
-		}
-
 		if err != nil {
 			log.Warnf("could not get workspaces: %v", err)
+		} else {
+			if len(resp.GetStatus()) == 0 {
+				break
+			}
```

Comment:
It address address the https://github.com/gitpod-io/gitpod/pull/9940#discussion_r874312688